net/scm/xscm_proxy.c

```diff
@@ -198,7 +198,6 @@ static void xaprc00x_proxy_process_transmit(struct work_struct *work)
 		break;
 	}
 }
-EXPORT_SYMBOL_GPL(scm_proxy_recv_ack);
```

Comment:
Why do only a subset of the proxy functions need to be exported?

Reply:
That's all the USB driver needs. It used to be organized that the USB driver figures out what kind of packet it had and calls the appropriate function in the proxy. I moved that function in here so all USB to lower the number if unique calls and remove any packet logic from the USB driver. I could export more but I'm not sure of the use.

Reply:
:+1: